com.unity.netcode.gameobjects/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -407,6 +407,14 @@ public void TrackSceneEventReceived(ulong senderClientId, uint sceneEventType, s
             IncrementMetricCount();
         }
 
+        public void TrackPacketSent(uint packetCount)
+        {
+        }
+
+        public void TrackPacketReceived(uint packetCount)
+        {
+        }
+
```

Comment:
Is this waiting for some changes in the tools package?

Reply:
This will be part of the proper JIRA Ticket that dispatch it. With tests and all. It's there just to make sure we have functions.